src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -3201,4 +3201,7 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="No_valid_statement_range_to_extract" xml:space="preserve">
     <value>No valid statement range to extract</value>
   </data>
+  <data name="Convert_to_field_property" xml:space="preserve">
+    <value>Convert to 'field' property</value>
```

Comment:
Consider adding a `<comment>{Locked="field"} "field" is C# keyword and should not be localized.</comment>`

Reply:
Good idea!